### PR TITLE
Provost Rankpansion

### PR DESCRIPTION
**Pull request by ItsVyzo**
# About the pull request
This added the missing Provost positions from the Provost Office hierarchy and gives more options for a more appropriate response when handling ML issues ICly including a Deputy Marshal and Provost Chief Inspector.

it keeps conflicting every single sprite merge 🗡️ 

# Explain why it's good for the game
rank under-response is better than rank over-response for roleplay and immersion

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

image
dreamseeker_Wqypw6erPa
dreamseeker_YLyXXv71vc
dreamseeker_rIGx5diI14
dreamseeker_ivs408zuvU

</details>

# Changelog

:cl:
add: Provost Chief Inspector
add: missing Provost Presets per ML Hierarchy
imageadd: added PCI sprites
imageadd: PCI and Dep. Marshal HUD icons
/:cl:

### CODE REVIEW

File: code/datums/paygrades/factions/uscm/provost.dm [hunk 1]
```diff
@@ -10,6 +10,19 @@
...
+	paygrade = PAY_SHORT_PVCI
+	name = "Provost Chief Inspector"
+	prefix = "Chief Insp."
+	rank_pin = /obj/item/clothing/accessory/ranks/special/insp
+	officer_grade = GRADE_FLAG //Not really a flag officer, but they have special access to things for their job.
+
+/datum/paygrade/provost/marshal/deputy
+	paygrade = PAY_SHORT_PVDM
+	name = "Provost Deputy Marshal"
+	prefix = "Deputy Marshal"
```
Comment (realforest2001) [inline code/datums/paygrades/factions/uscm/provost.dm:23]:
```suggestion
	prefix = "Dep. Marshal"
```

File: code/modules/clothing/under/marine_uniform.dm [hunk 2]
```diff
@@ -381,6 +381,12 @@
 	icon_state = "provost_tml"
 	worn_state = "provost_tml"
 
+/obj/item/clothing/under/marine/mp/provost/chief
+	name = "\improper Chief Provost Uniform"
+	desc = "The crisp uniform of a chief member of the Provost Office."
```
Comment (realforest2001) [inline code/modules/clothing/under/marine_uniform.dm:386]:
```suggestion
	name = "\improper Provost Command Uniform"
	desc = "The crisp uniform of a commanding member of the Provost Office."
```

File: code/modules/clothing/suits/marine_coat.dm [hunk 3]
```diff
@@ -243,6 +243,11 @@
 	flags_atom = NO_SNOW_TYPE|NO_NAME_OVERRIDE
 	valid_accessory_slots = list(ACCESSORY_SLOT_ARMBAND, ACCESSORY_SLOT_RANK, ACCESSORY_SLOT_DECOR)
 
+/obj/item/clothing/suit/storage/jacket/marine/provost/chief
+	name = "\improper Chief Provost Jacket"
```
Comment (realforest2001) [inline code/modules/clothing/suits/marine_coat.dm:247]:
```suggestion
	name = "\improper Provost Command Jacket"
```

File: code/modules/clothing/head/head.dm [hunk 4]
```diff
@@ -489,6 +489,11 @@
 	icon_state = "coblackberet"
 	desc = "A beret with the USCM Military Police insignia emblazoned on it."
 
+/obj/item/clothing/head/beret/marine/mp/provost/chief
+	name = "\improper USCM chief provost beret"
```
Comment (realforest2001) [inline code/modules/clothing/head/head.dm:493]:
```suggestion
	name = "\improper USCM provost command beret"
```

File: code/modules/clothing/head/head.dm [hunk 5]
```diff
@@ -489,6 +489,11 @@
...
+	icon_state = "pvciberet"
+	desc = "A beret with the USCM Military Police insignia emblazoned on it."
```
Comment (realforest2001) [inline code/modules/clothing/head/head.dm:495]:
```suggestion
```
Doesn't need an independent definition of desc as it inherits an identical one already.

**Review by realforest2001 — CHANGES_REQUESTED** [review]

File: code/datums/factions/uscm.dm [hunk 6]
```diff
@@ -142,10 +142,16 @@
...
+				marine_rk = "pvci"
+				border_rk = "command"
 			if(JOB_PROVOST_ADVISOR)
 				marine_rk = "pva"
 				border_rk = "command"
-			if(JOB_PROVOST_MARSHAL, JOB_PROVOST_CMARSHAL, JOB_PROVOST_SMARSHAL)
+			if(JOB_PROVOST_DMARSHAL)
+				marine_rk = "pvdm"
+				border_rk = "command"
+			if(JOB_PROVOST_MARSHAL, JOB_PROVOST_CMARSHAL, JOB_PROVOST_FMARSHAL, JOB_PROVOST_SMARSHAL)
```
Comment (realforest2001) [inline code/datums/factions/uscm.dm:154]:
```suggestion
			if(JOB_PROVOST_MARSHAL, JOB_PROVOST_CMARSHAL, JOB_PROVOST_SMARSHAL)
```

File: code/datums/paygrades/factions/uscm/provost.dm [hunk 7]
```diff
@@ -22,6 +35,12 @@
 	prefix = "S. Marshal"
 	officer_grade = GRADE_FLAG
 
+/datum/paygrade/provost/fleetmarshal
+	paygrade = PAY_SHORT_PVFM
+	name = "Provost Fleet Marshal"
+	prefix = "Fleet Marshal"
+	officer_grade = GRADE_FLAG
+
```
Comment (realforest2001) [inline code/datums/paygrades/factions/uscm/provost.dm:43]:
```suggestion
```

File: code/game/jobs/job/special/provost.dm [hunk 8]
```diff
@@ -22,6 +30,10 @@
 /datum/job/special/provost/marshal/sector
 	title = JOB_PROVOST_SMARSHAL
 
+//Provost Chief Marshal
+/datum/job/special/provost/marshal/fleet
+	title = JOB_PROVOST_FMARSHAL
+
```
Comment (realforest2001) [inline code/game/jobs/job/special/provost.dm:36]:
```suggestion
```

File: code/modules/cm_marines/marines_consoles.dm [hunk 9]
```diff
@@ -925,9 +925,12 @@ GLOBAL_LIST_EMPTY_TYPED(crewmonitor, /datum/crewmonitor)
 				JOB_PROVOST_CMARSHAL = 00,
 				JOB_GENERAL = 00,
 				JOB_PROVOST_SMARSHAL = 01,//Grade O9
-				JOB_PROVOST_MARSHAL = 02,//Grade O8
+				JOB_PROVOST_FMARSHAL = 02,//Grade O8
+				JOB_PROVOST_MARSHAL = 03,//Grade O7
+				JOB_PROVOST_DMARSHAL = 04,//Grade O6
```
Comment (realforest2001) [inline code/modules/cm_marines/marines_consoles.dm:930]:
```suggestion
				JOB_PROVOST_MARSHAL = 02,//Grade O7
				JOB_PROVOST_DMARSHAL = 03,//Grade O6
```

File: code/modules/gear_presets/uscm_event.dm [hunk 10]
```diff
@@ -374,14 +425,23 @@
...
 
+/datum/equipment_preset/uscm_event/provost/marshal/fleet
+	name = "Provost Fleet Marshal (MO9)"
+	minimum_age = 50
+
+	assignment = JOB_PROVOST_FMARSHAL
+	rank = "Provost Fleet Marshal"
+	paygrades = list(PAY_SHORT_PVFM = JOB_PLAYTIME_TIER_0)
+	role_comm_title = PAY_SHORT_PVFM
+
```
Comment (realforest2001) [inline code/modules/gear_presets/uscm_event.dm:444]:
```suggestion
```

File: code/modules/gear_presets/uscm_event.dm [hunk 11]
```diff
@@ -374,14 +425,23 @@
 	new_human.equip_to_slot_or_del(new /obj/item/device/cotablet(new_human.back), WEAR_IN_BACK)
 
 /datum/equipment_preset/uscm_event/provost/marshal/sector
-	name = "Provost Sector Marshal (MO7)"
+	name = "Provost Sector Marshal (MO8)"
```
Comment (realforest2001) [inline code/modules/gear_presets/uscm_event.dm:428]:
```suggestion
	name = "Provost Sector Marshal (MO9)"
```

File: code/modules/gear_presets/uscm_event.dm [hunk 12]
```diff
@@ -327,6 +327,46 @@
 	new_human.equip_to_slot_or_del(new /obj/item/device/flash(new_human), WEAR_IN_JACKET)
 	new_human.equip_to_slot_or_del(new /obj/item/restraint/handcuffs(new_human), WEAR_IN_JACKET)
 
+/datum/equipment_preset/uscm_event/provost/inspector/chief
+	name = "Provost Chief Inspector"
+
+	assignment = JOB_PROVOST_CINSPECTOR
+	rank = "Provost Chief Inspector"
```
Comment (realforest2001) [inline code/modules/gear_presets/uscm_event.dm:334]:
```suggestion
	rank = JOB_PROVOST_CINSPECTOR
```

File: code/modules/gear_presets/uscm_event.dm [hunk 13]
```diff
@@ -336,8 +376,19 @@
 	role_comm_title = "PvA"
 	flags = EQUIPMENT_PRESET_EXTRA
 
+/datum/equipment_preset/uscm_event/provost/marshal/deputy
+	name = "Provost Deputy Marshal (MO6)"
+	minimum_age = 45
+	skills = /datum/skills/general
+
+	assignment = JOB_PROVOST_DMARSHAL
+	rank = "Provost Deputy Marshal"
```
Comment (realforest2001) [inline code/modules/gear_presets/uscm_event.dm:385]:
```suggestion
	rank = JOB_PROVOST_DMARSHAL
```

**Review by realforest2001 — CHANGES_REQUESTED** [review]

File: code/__DEFINES/job.dm [hunk 14]
```diff
@@ -331,11 +331,14 @@ GLOBAL_LIST_INIT(job_command_roles, JOB_COMMAND_ROLES_LIST)
 #define JOB_PROVOST_TML "Provost Team Leader"
 #define JOB_PROVOST_ADVISOR "Provost Advisor"
 #define JOB_PROVOST_INSPECTOR "Provost Inspector"
+#define JOB_PROVOST_CINSPECTOR "Provost Chief Inspector"
+
```
Comment (realforest2001) [inline code/__DEFINES/job.dm:335]:
```suggestion
#define JOB_PROVOST_CINSPECTOR "Provost Chief Inspector"
#define JOB_PROVOST_UNDERCOVER "Provost Undercover Inspector"

```

File: code/datums/factions/uscm.dm [hunk 15]
```diff
@@ -142,9 +142,15 @@
 			if(JOB_PROVOST_INSPECTOR)
 				marine_rk = "pvi"
 				border_rk = "command"
+			if(JOB_PROVOST_INSPECTOR, JOB_PROVOST_CINSPECTOR)
+				marine_rk = "pvci"
+				border_rk = "command"
```
Comment (realforest2001) [inline code/datums/factions/uscm.dm:147]:
```suggestion
			if(JOB_PROVOST_UNDERCOVER)
				marine_rk = "pvuc"
				border_rk = "command"
			if(JOB_PROVOST_CINSPECTOR)
				marine_rk = "pvci"
				border_rk = "command"
```

**Review by realforest2001 — APPROVED** [review]

**Review by Blundir — APPROVED** [review]

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Pull request merged**